shell/edit/provisioning.cattle.io.cluster/rke2.vue

```diff
@@ -1686,6 +1742,14 @@ export default {
         <Tab name="basic" label-key="cluster.tabs.basic" :weight="11" @active="refreshYamls">
           <Banner v-if="!haveArgInfo" color="warning" label="Configuration information is not available for the selected Kubernetes version.  The options available in this screen will be limited, you may want to use the YAML editor." />
           <Banner v-if="showk8s21LegacyWarning" color="warning" :label="t('cluster.legacyWarning')" />
+          <Banner
+            v-if="isHarvesterDriver && isHarvesterIncompatible"
```

Comment:
I think there should be `&& showCloudProvider` as well so the banner about cloud providers is not shown when the user cannot select a cloud provider (k3s kubernetes version is selected)